docs/source/user-guide/docker_files.md

```diff
@@ -22,27 +21,27 @@ Follow the [instruction](https://docs.github.com/en/packages/working-with-a-gith
 Note: if the docker image already exists in your system, this same image will be used to run the container. This might especially be an issue when using the `latest` tag, which might not be the latest version available in the registry. To force a pull of the latest image, use the `--pull` option, e.g.:
 
 ```bash
-docker run --pull always --rm -it ghcr.io/gammasim/simtools-prod:latest bash
+docker run --pull always --rm -it ghcr.io/gammasim/simtools-dev:latest bash
 ```
 
 Alternatively, delete the image first and pull the latest version from the registry.
 
-## Container for simtools users (simtools-prod)
+## Container for simtools users ('no opt' images)
 
 Provides a container for simtools users, which includes:
 
 - corsika and sim\_telarray
 - packages required by simtools
 - simtools (main branch)
 
-### Run a simtools-prod container
+### Run a simtools user container
 
 Prerequisite: configure the simulation model database access (see simtools documentation) similar to the [template example](https://github.com/gammasim/simtools/blob/main/.env_template).
 
 To startup a container to use bash
 
 ```bash
-docker run --rm -it --env-file .env -v "$(pwd):/workdir/external" ghcr.io/gammasim/simtools-prod:latest bash
+docker run --rm -it --env-file .env -v "$(pwd):/workdir/external" ghcr.io/gammasim/simtools-prod-240205-corsika-77500-bernlohr-1.67-prod6-baseline-qgs2-no_opt::latest bash
```

Comment:
There are two `:` before latest. Probably not an issue, but still.

Reply:
Fixed.